navigation/navigation-compose/src/nonAndroidMain/kotlin/androidx/navigation/compose/ComposeNavigator.nonAndroid.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.navigation.compose
+
+import androidx.compose.animation.AnimatedContentScope
+import androidx.compose.animation.AnimatedContentTransitionScope
+import androidx.compose.animation.EnterTransition
+import androidx.compose.animation.ExitTransition
+import androidx.compose.animation.SizeTransform
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavDestination
+import androidx.navigation.NavOptions
+import androidx.navigation.Navigator
+import androidx.navigation.compose.ComposeNavigator.Destination
+import androidx.navigation.implementedInJetBrainsFork
+import kotlin.jvm.JvmSuppressWildcards
+import kotlinx.coroutines.flow.StateFlow
+
+public actual class ComposeNavigator actual constructor() : Navigator<Destination>() {
+
+    internal actual val transitionsInProgress: StateFlow<Set<NavBackStackEntry>>
+        get() = implementedInJetBrainsFork()
```

Comment:
All `implementedInJetBrainsFork` must be removed

Reply:
oops